azure/template.json

```diff
@@ -138,7 +145,15 @@
     "keyvaultCertificateName": "[if(greater(length(parameters('certificateName')),0), parameters('certificateName'), replace(parameters('customHostName'), '.', '-'))]",
     "appServiceName": "[concat(variables('resourceNamePrefix'), '-as')]",
     "appServicePlanName": "[concat(variables('resourceNamePrefix'), '-asp')]",
-    "appServiceRuntimeStack": "[concat('DOCKER|', parameters('dockerHubUsername'), '/', parameters('containerImageReference'))]"
+    "appServiceRuntimeStack": "[concat('DOCKER|', parameters('dockerHubUsername'), '/', parameters('containerImageReference'))]",
+    "frontDoorName": "[concat(variables('resourceNamePrefix'),'-fd')]",
+    "frontDoorAppServiceName": "[concat(variables('appServiceName'), '.azurewebsites.net')]",
```

Comment:
This appears only one app service can be in the backend pool. Can it be made so more than one app service can be in the backend, eg. an array variable of backend app services